src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -107,14 +107,14 @@ public function getFormErrorsForJS(FormInterface $form)
         $translator = $this->get('translator');
 
         foreach ($form->getErrors(true) as $error) {
-            if (!$error->getCause()) {
-                $formId = 'bubbling_errors';
-            } else {
+            if ($error->getCause() && method_exists($error->getCause(), 'getPropertyPath')) {
```

Comment:
Same as above. What happened there?

Reply:
This one I don't know either

Reply:
Dont remember now how to reproduce it but remember that there was case where $error->cause was not null but it didnt had getPropetyPath and because of that the str_replace would break. It just adds extra check to make sure that function inside can be completed without throwing exception